cmd/rollouts-controller/main.go

```diff
@@ -283,6 +291,7 @@ func newCommand() *cobra.Command {
 	command.Flags().IntVar(&klogLevel, "kloglevel", 0, "Set the klog logging level")
 	command.Flags().IntVar(&metricsPort, "metricsport", controller.DefaultMetricsPort, "Set the port the metrics endpoint should be exposed over")
 	command.Flags().IntVar(&healthzPort, "healthzPort", controller.DefaultHealthzPort, "Set the port the healthz endpoint should be exposed over")
+	command.Flags().IntVar(&pprofPort, "pprof-port", controller.DefaultPProfPort, "Set the port the pprof endpoint should be exposed over")
```

Comment:
In my mind as a user, I would like to be able to overwrite the port in case I'm using the default debug port `6060` elsewhere. I will defer on a maintainer's judgement for what pattern makes the most sense in the project given this is creating two flags for one feature.

Reply:
I am kind of a fan of passing in a single flag of such that we can do `0.0.0.0:6060` or `localhost:6062` . The main reason for this is not all pods can you get into and it might be nice to be a be able to remotely pull pprof files if configured.

Reply:
In terms of wording, a boolean is much easier to convey `--enable-pprof=true` vs. `--pprof-port` being the boolean toggle _and_ the config for the address. Do you have any ideas on how we can convey that? 

I see two options with tradeoffs:

1. Having `--pprof-port` resolve to an address doesn't make it clear that this is disabled by default (maybe the help text is what we need to clarify here).
2. Setting `--enable-pprof` equal to an address instead of a boolean feels strange to me. Maybe something like `--enable-pprof-server-address` is a (verbose) happy medium?

Reply:
Yea, thats hard. We could stick with the status quo with the code base and be a little less secure which uses [0.0.0.0](https://github.com/argoproj/argo-rollouts/blob/master/controller/controller.go#L100) by default and then just have the enable on port flag.

I don't like the above due to the security concerns around the data available on the pprof endpoints. So with that said I think let's go verbos with `--enable-pprof-server-address` or `--enable-pprof-address`  I think I am impartial on weather `server` is there or not, thoughts?

Reply:
I pushed up a commit to consolidate the two config vars. My remaining question is what you want to do about the default value? We obviously don't want to enable profiling for every user so I don't know know a way around `""` as the default unless you have suggestions?

Reply:
What you did for default is perfect, this LGTM thanks for adding this.